sqlparser_bench/src/lib.rs

```diff
@@ -0,0 +1,7 @@
+#[cfg(test)]
```

Comment:
Was this intended to be committed?

Reply:
Good catch, is the result of running `cargo new --lib`, didn't notice that.

Reply:
Thanks @maxcountryman! I removed this file from master.